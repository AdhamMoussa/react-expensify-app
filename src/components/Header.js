import React from 'react';
import { NavLink } from 'react-router-dom';

const Header = () => {
  return (
    <div>
      <h1>Expensify</h1>
      <nav>
        <NavLink to='/' activeClassName="active-link" exact={true}>DashBoard</NavLink>
        <NavLink to='/create' activeClassName="active-link">Add Expense</NavLink>
        <NavLink to='/help' activeClassName="active-link">Help</NavLink>
      </nav>
    </div>
  )
};

export default Header;
