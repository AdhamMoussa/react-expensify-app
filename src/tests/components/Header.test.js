import React from 'react';
import Header from '../../components/Header';
import { shallow } from 'enzyme';

test('should render Header correctly', () => {
  const wrapper = shallow(<Header />);
  expect(wrapper).toMatchSnapshot();
});