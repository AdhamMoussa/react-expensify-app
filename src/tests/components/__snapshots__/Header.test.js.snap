// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<div>
  <h1>
    Expensify
  </h1>
  <nav>
    <NavLink
      activeClassName="active-link"
      aria-current="page"
      exact={true}
      to="/"
    >
      DashBoard
    </NavLink>
    <NavLink
      activeClassName="active-link"
      aria-current="page"
      to="/create"
    >
      Add Expense
    </NavLink>
    <NavLink
      activeClassName="active-link"
      aria-current="page"
      to="/help"
    >
      Help
    </NavLink>
  </nav>
</div>
`;
